### PR TITLE
rwt copy fix for Windows

### LINKED ISSUE #679: yarn rwt copy depends on rsync, which isn't available on Windows
redwood@d51ade08118c17459cebcdb496197ea52485364a
windows 10
GitBash (mintty 3.1.6)
yarn v1.22.4

I tried copying over a modified version of the Redwood Framework to a Redwood App, to test some changes I've made. But trying to run `yarn rwt copy` I get an error message about rsync not being available. Which makes sense, since I'm on Windows. See log below

```
$ yarn rwt copy ../redwood
yarn run v1.22.4
$ C:\Users\tobbe\dev\redwood\test-store\node_modules\.bin\rwt copy ../redwood
Redwood Framework Path:  C:\Users\tobbe\dev\redwood\redwood
'rsync' is not recognized as an internal or external command,
operable program or batch file.
(node:35440) UnhandledPromiseRejectionWarning: Error: Command failed with ENOENT: rsync -rtvu --delete 'C:\Users\tobbe\dev\redwood\redwood/packages/' 'C:\Users\tobbe\dev\redwood\test-store/node_modules/@redwoodjs/'
spawn rsync ENOENT
    at notFoundError (C:\Users\tobbe\dev\redwood\test-store\node_modules\@redwoodjs\cli\node_modules\cross-spawn\lib\enoent.js:6:26)
    at verifyENOENT (C:\Users\tobbe\dev\redwood\test-store\node_modules\@redwoodjs\cli\node_modules\cross-spawn\lib\enoent.js:40:16)
    at ChildProcess.cp.emit (C:\Users\tobbe\dev\redwood\test-store\node_modules\@redwoodjs\cli\node_modules\cross-spawn\lib\enoent.js:27:25)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
(node:35440) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:35440) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 1.23s.
```

### PR DESCRIPTION
**Pull request by Tobbe**
Fixes #679

### CODE REVIEW

**Comment by Tobbe** [comment]
Need to make sure this works with PowerShell as well before merging

File: packages/cli/src/redwood-tools.js [hunk 1]
```diff
@@ -66,6 +66,16 @@ export const fixProjectBinaries = (PROJECT_PATH) => {
 
 export const copyFiles = async (src, dest) => {
   // TODO: Figure out if we need to only run based on certain events.
+
+  if (process.platform === 'win32') {
+    // rsync doesn't do Windows paths, so we need to change them to posix paths
```
Comment (peterp) [inline packages/cli/src/redwood-tools.js:71]: lol, amazing

Comment (Tobbe) [reply packages/cli/src/redwood-tools.js:71]: 😁

**Comment by thedavidprice** [comment]
Changes to CONTRIBUTING.md with link to blog post == ✅ Thanks @Tobbe!

@peterp I leave this one to you for merging.

**Pull request merged**